packages/odie-client/src/components/message/get-support.tsx

```diff
@@ -68,7 +68,7 @@ export const GetSupport: React.FC< GetSupportProps > = ( {
 		const buttons: ButtonConfig[] = [];
 
 		if ( isPaidUser ) {
-			if ( isEmailForced ) {
+			if ( isEmailForced || ! isChatLoaded ) {
```

Comment:
I believe this breaks GetSupport for users providing negative feedback on articles, they will always see "Send an email" instead of going to talk to AI. Fixing it here https://github.com/Automattic/wp-calypso/pull/107894